### PR TITLE
[Merged by Bors] - Added keyboard key to mouse control for scene viewer example

### LINKED ISSUE #4407: Add secondary keybinding to "move camera" in `scene_viewer` example
## What problem does this solve or what need does it fill?

On some touchpads it can be challenging to both click and pan simultaneously, making it onerous to pan the camera in the `scene_viewer` example following #4405.

## What solution would you like?

Add another keybinding (in addition Left Mouse Button) to enable camera movement. @superdump suggested `KeyCode::L`, which I like.

## Additional context

https://github.com/bevyengine/bevy/pull/4405#issuecomment-1087032550

### PR DESCRIPTION
**Pull request by Sheepyhead**
# Objective

- Added keyboard control for scene_viewer example. Fixes #4407

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

File: examples/tools/scene_viewer.rs [hunk 1]
```diff
@@ -455,7 +457,9 @@ fn camera_controller(
 
         // Handle mouse input
         let mut mouse_delta = Vec2::ZERO;
-        if mouse_button_input.pressed(options.key_enable_mouse) {
+        if mouse_button_input.pressed(options.mouse_key_enable_mouse)
+            || key_input.pressed(options.keyboard_key_enable_mouse)
```
Comment (superdump) [inline examples/tools/scene_viewer.rs:461]: This approach doesn't work too well as you would have to hold down M while using WASD. I think it is better to make the key press a toggle as to whether mouse look is enabled or not. You could use a `Local<bool>` in the camera controller system perhaps.

**Review by superdump — CHANGES_REQUESTED** [review]

File: examples/tools/scene_viewer.rs (hunk 1 above)
Comment (alice-i-cecile) [reply examples/tools/scene_viewer.rs:461]: IMO we should just use a resource for this, but the UX argument makes sense.

Comment (Sheepyhead) [reply examples/tools/scene_viewer.rs]: I've implemented that solution now, it seems to work very well!

**Review by alice-i-cecile — APPROVED** [review]

**Review by superdump — APPROVED** [review]
Resolve the conflicts and we can get this merged.

**Comment by Sheepyhead** [comment]
> Resolve the conflicts and we can get this merged.

Done!

**Pull request closed without merging**